### ISSUE TITLE
Enhance reusability of findingaids by extracting a Rails engine

### ISSUE THREAD

**Issue by barmintor** [NYULibraries/findingaids#125]
Our local fork of findingaids is being pretty well-received, but it has a bunch of configuration changes and string content replacements that suggest an engine could be extracted (to minimize the difficulty of staying up-to-date). It would also more cleanly leave authorization dependencies and EAD url resolution (as well as the window dressing of page names) to the apps built on the engine.

Is there interest in that on the NYU end? Is there interest in organizing a brief buildathon for it in NYC, perhaps under the auspices of Code4Lib/METRO? For what it's worth, there doesn't appear to be a gem called FABLE in rubygems. I'm just saying, FindingAidsBlackLightEngine.

**Comment by sallyvermaaten** [comment]
Hey Ben, 

Great to hear the findingaids code is looking useful. Making the code more re-usable by extracting an engine to separate some of local configurations and dependencies sounds like a really good idea. My feeling is that there will be interest here in some collaboration - let me check in with folks here. It will probably take me a few days to confirm as several folks are out on vacation this week. 

Also, it's probably relevant to mention the status of our overall archival / special collections discovery work - which we split into 2 phases. We worked on findingaids last year as Phase I - a relatively quick, tightly scoped replacement for our previous, legacy archival search interface which was pretty old. Phase II - which we're about to kick off in September- is meant to be wider re-think of our needs around archival discovery, starting with some design work and conversations about what improvements we can make to best help users. We wanted to start Phase II after we implemented ArchivesSpace locally, in part because we're thinking we might take advantage of the ASpace API. Happy to share more info / talk further about this if you're interested! 

More soon, 
Sally
